tools/docker/sles.docker

```diff
@@ -23,7 +23,12 @@ RUN zypper --gpg-auto-import-keys install -y \
     gdb \
     git \
     python3-pip \
-    rpm-build
+    rpm-build \
+    perl-File-BaseDir \
+    libgfortran5 \
+    hipblaslt-devel \
+    hipblas-devel
```

Comment:
These should also be added to the install_preqs.

Reply:
Done.